test/influx-integration.spec.js

```diff
@@ -444,6 +444,34 @@ describe('@integration influxdb tests', () => {
             });
         });
 
+        describe('lag', () => {
+            it('successfully retrieves lagging points', () => {
+                // Start inserting points every 0.1s with 0.5s timestamp lag
```

Comment:
A certain margin wouldn't hurt, but I think that same numbers make the test easier to understand. If we see 'random' failures, then lets increase this.